source/pastemyst/db/db.d

```diff
@@ -80,6 +80,19 @@ public void insert(T)(T item)
     collection.insert(data);
 }
 
+/++
+ + Get the amount of Paste objects in the DB
+ +/
+public ulong getPastiesAmount(T)()
+{
+    MongoCollection collection = mongo[getCollectionName!T()];
+
+    // this here might not be the best solution
+    ulong count = collection.count ("");
+
+    return count;
```

Comment:
you can just do `return collection.count("")` here. also no need for that comment, this is a nice solution (although a bit ugly but that is because `.count` requires a parameter for some reason).